### ISSUE TITLE
Wasn't able to parse friends index

### ISSUE THREAD

**Issue by Laura2797** [jcontini/facebook-scraper#22]
Hello, this is my first time using your app. I was trying to get the friends list but, the next error message appers when i execute the next command:
$ python make.py --index

Loaded credentials from .env file.
Opening saved friends list...

Wasn't able to parse friends index. This probably means that Facebook updated their template. 
Please raise issue on Github and I will try to update the script. 
Or if you can code, please submit a pull request instead :)

I tried to use the command --download and -- parse, but it return a empty list in json.

Thank you

**Comment by jcontini** [comment]
Hi @Laura2797,

For the first run, try running just `python make.py` without any of the parameters. Currently that's the only time that it will download the friend list. I'll clarify that in the readme for the next update.

If that doesn't work, try commenting out this line: `options.add_argument('--headless')`, so that it's `#options.add_argument('--headless')`. This should make Chrome visible so you can see what is happening and where it might be getting stuck. 

I just pushed an update (15 minutes after you posted) with a few fixes. Though I don't know if it'll fix this particular issue, can you download the latest one just to be sure?

**Comment by Laura2797** [comment]
Hello @jcontini, thank you for you very fast answer! 

I tried run "python make.py" and it create a new file index.html. It has a html code, but the index.json and profiles.json was empty.
Later I tried comment the line options.add_argument('--headless'), and i saw that the chrome open a new window and then go to the friend tab, but dont get anymore information. In the terminal show the same message than before.
And finally I update the repository with your fixes, and creates a file "friend_list.html" and data.db. The .db is empty too, but I'm not sure.

In conclusion, the .html have information, but the .json and .db are empty.

Anyways, I'm interested that the information returns in a .json format is that possible?

Thank you, have a good day!

**Comment by jcontini** [comment]
Hi @Laura2797,

If you open the `friend_list.html` file on your computer, can you see the list of your friends? 

Ah it's good to know there's an appetite for the JSON format after all. I'll update the script later to add a feature for JSON export.

Either way, would you be comfortable sending me the file (github@contini.co) so I can see if it parses on my computer and try to troubleshoot? Or if you prefer, we can do a screenshare session instead via Zoom/Google Meet.

**Comment by Laura2797** [comment]
Hi @jcontini, I just send you the email to github@contini.co, notice me if you dont recieve the email.

**Comment by jcontini** [comment]
Hi @Laura2797, thanks - helping you out there. Also, I've just added a feature for JSON export. To use it, just run `python make.py --json`. This will create 3 json files (friend_list, profiles, locations) in the db/ folder. Let me know if it works for you, or if you're looking for different data in the files.
